x/account/types/genesis_test.go

```diff
@@ -0,0 +1,98 @@
+package types
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGenesisStateValidate(t *testing.T) {
+	tests := []struct {
+		name     string
+		genState *GenesisState
+		err      error
+	}{
+		{
+			name:     "default is valid",
+			genState: DefaultGenesis(),
+		}, {
+			name: "valid custom genesis",
+			genState: &GenesisState{
+				CoordinatorByAddressList: []*CoordinatorByAddress{
+					{CoordinatorId: 10, Address: "cosmos1c7gh3kejxm3pzl8fwe65665xncs24x5rl7a8sm"},
```

Comment:
The prefix for the addresses on Starport Network are actually `spn` instead of `cosmos`

Reply:
Also we may have many tests in the future that necessitate to use addresses. Maybe we can add helpers in `testutil` package to provide the address.

We used to use this method in precious spn version to get a randomized address: https://github.com/tendermint/spn/blob/cd4de2f915a75e8fd67b2dfeb1e4c023fbb1349f/internal/testing/mock.go#L37

Reply:
I like the idea of having some fixtures for future tests, not only addresses but coordinators, chain, validators...

Reply:
What do you mean by fixtures?

Reply:
sorry! fixtures are the same of the create some data for the tests, but the name is more used in the Python language

Reply:
Ok I didn't know👍 FYI I created such a package in another PR https://github.com/tendermint/spn/blob/feat/init-genesis-account/testutil/sample/sample.go